Islandora-Devops/claw-playbook
# Remove Grok.

**GitHub Issue**: [#2166 \[CODE TASK\] Playbook takes a very long time. ](https://github.com/Islandora/documentation/issues/2166)

# What does this Pull Request do?

Removes Grok from the playbook since it is no longer needed.

# What's new?

For a while now the playbook has been installing OpenJpeg from the Ubuntu repository. That makes this task redundant since it was disguising its executable as opj_decompress anyway.

* Does this change require documentation to be updated? No 
* Does this change add any new dependencies?  No
* Does this change require any other modifications to be made to the repository 
 (i.e. Regeneration activity, etc.)?  No
* Could this change impact execution of existing code? No

# How should this be tested?

Build a playbook on this branch and ensure that JP2 files are handled as expected.

Specifically:

## Add test objects:
1. Enable the Islandora Mirador module.
2. Create a Paged Content object with display hint 'Mirador'
3. Add a child object with model Page.
4. Add a File media to that object and upload a JP2 file. Check the "Original File" role.

## Configure Mirador 
1. Go to Admin -> Index and click on "Context" under the Context UI category
2. Edit the "Paged Content - Mirador" context.
3. There should already be a Block reaction but there are no blocks placed yet. Click "Place Block"
4. Choose the Mirador block, place it in Content.
5. In the Manifest URL text field, put https://localhost:8000/node/[node:nid]/book-manifest-original
6. Save the block, then click "Save and Continue"

## Verify JP2 behaviour
1. Go to the child object and click on the Media tab. 
2. You should see Thumbnail and Service File media generated from the original JP2 File media
3. Go to the parent object. You should see a Mirador interface, and you can verify by right-clicking or via Web Inspector that it is making requests to Cantaloupe for slices of a JP2 image.

# Additional Notes:

This saves about 5 minutes  from the playbook build process, and removes a long-standing dependency on software that we were not updating.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora-Devops/committers @rosiel
